l5-operator/config/resources/bestie-deploy.yaml

```diff
@@ -36,7 +33,6 @@ spec:
           value: app
         - name: FLASK_ENV
           value: development
-          #todo fixme
```

Comment:
should leave this in we should not have secrets in git

Reply:
I added it back